app/view_utils/topbar_helper.rb

```diff
@@ -127,4 +129,8 @@ def missing_profile_image_path
   def paths
     @_url_herlpers ||= Rails.application.routes.url_helpers
   end
+
+  def link_external?(url, host_with_port)
```

Comment:
Yeah, but `FeatureFlagHelper` has `feature_enabled?` which takes a feature and `Email` has `email_available?` which takes an email so I planned to follow the naming on those.